app/Helpers/Helper.php

```diff
@@ -43,52 +43,50 @@ function number($number, $decimals=2) {
  * @return array with `hours`, `minutes` and `showHours`.
  */
 function secondsToDuration($seconds): array {
-    $secondsInAnHour = 60*60;
+    $secondsInAnHour = 60 * 60;
 
     return [
-        "hours" => intdiv($seconds, $secondsInAnHour),
-        "minutes" => intdiv($seconds % $secondsInAnHour, 60),
+        "hours"     => intdiv($seconds, $secondsInAnHour),
+        "minutes"   => intdiv($seconds % $secondsInAnHour, 60),
         "showHours" => $seconds >= $secondsInAnHour
     ];
 }
 
 /**
  * @param array $duration from the secondsToDuration
+ * @return string
  */
-function durationToSpan($duration): String {
+function durationToSpan($duration): string {
     $return = $duration["minutes"] . "<small>min</small>";
 
-    if($duration["showHours"]) {
+    if ($duration["showHours"]) {
         $return = $duration["hours"] . "<small>h</small>&nbsp;" . $return;
     }
 
     return $return;
 }
 
-function stationLink($name, $classes = "text-trwl clearfix"): String {
+function stationLink($name, $classes = "text-trwl clearfix"): string {
     $urlname = $name;
 
-    switch($name) {
+    switch ($name) {
         // Those are stations that you can ride to but you can't search for them.
         case $name == "Köln Messe/Deutz Gl. 9-10":
         case $name == "Köln Messe/Deutz Gl.11-12":
             $urlname = "Köln Messe/Deutz";
-        break;
+            break;
 
         // Hamburg's Landungsbrücken has three bridges [1..3], but you cannot search for them.
         case preg_match('/Landungsbr.*cken Br.*cke \d/i', $name) > 0:
             $urlname = "Landungsbrücken, Hamburg";
-        break;
+            break;
     }
 
-    $return = '<a href="' . route('trains.stationboard') . '?provider=train&station=' . urlencode($urlname) . '" class="' . $classes . '">' . $name . '</a>';
-
-    return $return;
+    return '<a href="' . route('trains.stationboard') . '?provider=train&station=' . urlencode($urlname) .
+        '" class="' . $classes . '">' . $name . '</a>';
 }
 
-function formatNewDay($DateObject) {
-
-
-
-    return __("dates." . $DateObject->format('l')) .', '. $DateObject->format('j').'. '.__("dates." . $DateObject->format('F')) .' '. $DateObject->format('Y');
+function formatNewDay($dateObject) {
+    return __("dates." . $dateObject->format('l')) . ', ' . $dateObject->format('j') . '. '
```

Comment:
Why not using dateformatting for the whole date?

Would it not be possible like that?
```suggestion
    return __("dates." . $dateObject->format('l, j. F Y'));
```

Reply:
Thanks! This is a good idea.
I just looked up, where the function was used and saw it's just one usage and I replaced this with Carbon. So i removed it now: https://github.com/Traewelling/traewelling/pull/124/commits/3f856f5a3b21f6b05fd34768acf11dda6be9ee1b